iree/turbine/kernel/compiler/host_codegen.py

```diff
@@ -29,6 +31,10 @@
 def memref_to_tensor(memrefs: list[IrType]):
     tensors = []
     for m in memrefs:
+        # append scalars as-it-is to tensors list
+        if isinstance(m, (F32Type, IntegerType)):
```

Comment:
any thoughts on replacing `IntegerType` here  with `IntegerType.get_signless(32)`, in case someone feeds non 32 bit integer, we'd want to catch that